pkg/apis/helm.k0sproject.io/v1beta1/chart_types.go

```diff
@@ -43,6 +45,13 @@ func (cs ChartSpec) YamlValues() map[string]interface{} {
 	return CleanUpGenericMap(res)
 }
 
+// HashValues returns hash of the values
+func (cs ChartSpec) HashValues() string {
+	h := sha256.New()
+	h.Write([]byte(cs.ReleaseName + cs.Values))
+	return fmt.Sprintf("%x", h.Sum(nil))
```

Comment:
You could one-line this with `sha256.Sum256(..)` if you wanted. Not a deal breaker though.